controllers/pipeline/pipeline_adapter.go

```diff
@@ -202,6 +226,16 @@ func (a *Adapter) getImagePullSpecFromPipelineRun(pipelineRun *tektonv1beta1.Pip
 	return fmt.Sprintf("%s@%s", strings.Split(outputImage, ":")[0], imageDigest), nil
 }
 
+// getImagePullSpecFromPipelineRun gets the full image pullspec from the given ApplicationSnapshot Component,
```

Comment:
```suggestion
// getImagePullSpecFromSnapshotComponent gets the full image pullspec from the given ApplicationSnapshot Component,
```

Reply:
This is just a minor issue. But don't why this comment is marked `Resolved` by me when I approve it.

Reply:
Good catch, will fix that

Reply:
Fixed the description.